lib/rdoc/store.rb

```diff
@@ -197,6 +197,19 @@ def add_file absolute_name, relative_name: absolute_name, parser: nil
     top_level
   end
 
+  ##
+  # Make sure any references to C variable names are resolved to the corresponding class.
+  #
+
+  def resolve_c_references
```

Comment:
Will we need to resolve different types of references in the future? If we don't, `resolve_c_superclasses` may be better.

Reply:
I'm not sure what else will end up with C variable names instead of classes, so I named it to be open-ended. But I'm happy to give it a narrower name (we can always rename it if we find we need to resolve more references later).